plugins/arElasticSearchPlugin/lib/model/arElasticSearchInformationObject.class.php

```diff
@@ -65,11 +85,12 @@ public function recursivelyAddInformationObjects($parentId, $totalRows, $options
         // Descend hierarchy
         if (1 < ($item->rgt - $item->lft))
         {
-          // Pass ancestors, repository and creators down to descendants
+          // Pass ancestors, repository creators and term lookup table down to descendants
           $this->recursivelyAddInformationObjects($item->id, $totalRows, array(
             'ancestors'  => array_merge($node->getAncestors(), array($node)),
             'repository' => $node->getRepository(),
-            'inheritedCreators' => array_merge($node->inheritedCreators, $node->creators)));
+            'inheritedCreators' => array_merge($node->inheritedCreators, $node->creators),
+            'terms' => $options['terms']));
```

Comment:
Again, there seems to be no need to pass `option['terms']` as it's never updated.

Reply:
This will be changed as above in the latest PR.